doc/robot_model_and_robot_state/launch/robot_model_and_robot_state_tutorial.py

```diff
@@ -0,0 +1,39 @@
+import os
```

Comment:
Suggest renaming the file with a prefix of `.launch.py` to match the others

Reply:
you already did this

Reply:
done